### PR TITLE
Implemented AccountCreated event processing

### PR DESCRIPTION
**Pull request by mikemoody-amido**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/MTTL-1740.

## Describe this PR

* Implement processing for AccountCreated event
* Use new shared testing package
 
#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: HousingSearchListener.Tests/V1/E2ETests/Stories/UpdateAccountDetailsTests.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
+﻿using HousingSearchListener.Tests.V1.E2ETests.Fixtures;
+using HousingSearchListener.Tests.V1.E2ETests.Steps;
+using System;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace HousingSearchListener.Tests.V1.E2ETests.Stories
+{
+    [Story(
+        AsA = "SQS Tenure Listener",
```
Comment (humulla) [inline HousingSearchListener.Tests/V1/E2ETests/Stories/UpdateAccountDetailsTests.cs:10]:
```suggestion
        AsA = "SQS Housing Search Listener",
```

File: HousingSearchListener/V1/UseCase/Interfaces/IUpdateAccountDetailsUseCase.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace HousingSearchListener.V1.UseCase.Interfaces
+{
+    public interface IUpdateAccountDetailsUseCase : IMessageProcessing
+    {
```
Comment (humulla) [inline HousingSearchListener/V1/UseCase/Interfaces/IUpdateAccountDetailsUseCase.cs:4]: We need to add the methods here, not sure how your test is working with this empty
